app/assets/stylesheets/helpers/_layouts.scss

```diff
@@ -1,7 +1,3 @@
-#wrapper {
```

Comment:
I don't believe this change removes any styles attached to `#wrapper` - it's still defined using the `%site-width-container` mixin as before, it's just been moved around a little.

Originally I wanted to remove `#wrapper` entirely until I realised the massive impact this would have (which yes, we'd need to address in each application individually before merging this). I've stepped back from that, in that this change now removes the partial dependency on toolkit but still maintains the styles of  `#wrapper` as before.